spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -273,32 +273,36 @@ static void deleteExampleData(DatabaseClient dbClient) {
 
   // [START spanner_query_data]
   static void query(DatabaseClient dbClient) {
-    // singleUse() can be used to execute a single read or query against Cloud Spanner.
-    ResultSet resultSet =
+    // We use a try-with-resource block to automatically release resources held by ResultSet.
+    try (ResultSet resultSet =
         dbClient
```

Comment:
Is it really easiest to read with this on a seperate line from the = ?

Reply:
Updated.